src/lib.rs

```diff
@@ -214,6 +217,50 @@ impl Client {
         self.get_multi(keys).await
     }
 
+    // Gets the given key with additional metadata.
+    ///
+    /// If the key is found, `Some(Value)` is returned, describing the metadata and data of the key.
+    ///
+    /// Otherwise, `None` is returned.
+    ///
+    /// Supported meta flags:
+    /// - h: return whether item has been hit before as a 0 or 1
+    /// - l: return time since item was last accessed in seconds
+    /// - t: return item TTL remaining in seconds (-1 for unlimited)
+    pub async fn meta_get<K: AsRef<[u8]>>(
+        &mut self,
+        key: K,
+        meta_flags: Option<&[&str]>,
```

Comment:
`meta_flags` param here is a single optional slice of `&str`.  Could be swapped for `&[u8]` but I think the `&str` is fine in this case.  `Option<_>` is used because it's possible to perform a no-op on a key (providing no flags) to bump the LRU if desired.

One thing to consider is that this parameter would also include opaques.  It may be desirable to break the opaque token out as its own parameter.